lib/jekyll/converters/smartypants.rb

```diff
@@ -0,0 +1,21 @@
+module Jekyll
+  module Converters
+    class Smartypants < Converter
+      safe true
+
+      priority :lowest
+
+      def matches(ext)
+        true
+      end
+
+      def output_ext(ext)
+        ext
+      end
+
+      def convert(content)
+        content
```

Comment:
I'd like to just do any smarty stuff here. We don't need a bunch of different converters for it, right?